catalog/app/containers/Bucket/Queries/requests/requests.ts

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react'
+
+interface Cases<Data> {
+  Err?: (error: Error) => React.ReactNode
+  Init?: () => React.ReactNode
+  Ok?: (data: Data) => React.ReactNode
```

Comment:
there should be also a `Pending` case. also you can parameterize the output type